operator/config.go

```diff
@@ -33,3 +35,24 @@ func LoadConfig(path string) (*Config, error) {
 
 	return &config, nil
 }
+
+// envGanacheCmdVar is the environment variable to set the command
+// for spawning a `ganache-cli`. It is set "ERDSTALL_GANACHE_CMD".
+// See `ganacheCommand()`.
+const envGanacheCmdVar = "ERDSTALL_GANACHE_CMD"
```

Comment:
The `Var` is superfluous.
```suggestion
const envGanacheCmd = "ERDSTALL_GANACHE_CMD"
```